app/models/data/training_email_recipients.rb

```diff
@@ -0,0 +1,28 @@
+module Data
+  class TrainingEmailRecipients
+    include ToCsv
+    class << self
+
+      # @return [Array<String>]
+      def column_names
+        %w[Email]
+      end
+
+      # @return [Array<Hash{Symbol => Mixed}>]
+      def dashboard
+        User.training_email_recipients.map do |user|
```

Comment:
What if this scope is large? We should defend against it.

Reply:
Or are we confident there aren't many and this is a throw away bit of code?

Reply:
Good point. Now doing it in batches of 1000 [b3a1b43](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/736/commits/b3a1b434ec187732d92e027ffa16674fab2b9b68)

Reply:
I tried to answer my own question and `User.training_email_recipients` doesn't yet exist in `main`. Where is it coming from?